apps/src/code-studio/pd/workshop_dashboard/workshops/surveys/components/BarChartGroup.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  BodyTwoText,
+  StrongText,
+} from '@code-dot-org/component-library/typography';
+import {Box} from '@mui/material';
+import React from 'react';
+import {
+  BarChart,
+  Bar,
+  Cell,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Label,
+  LabelProps,
+  LabelList,
+  Rectangle,
+} from 'recharts';
+
+import moduleStyles from './barChartGroupStyles.module.scss';
```

Comment:
Why are some styles here but others specified below in the file?

Reply:
That's because `recharts` render charts into svg and I didn't find a way to override it's styles other then by using styles prop of `recharts` components